### PR TITLE
Update http_cache.rst

### PR DESCRIPTION
**Pull request by ifdattic**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.3 |
| Fixed tickets |  |

### CODE REVIEW

File: book/http_cache.rst [hunk 1]
```diff
@@ -856,7 +856,7 @@ If one content corresponds to one URL, the ``PURGE`` model works well.
 You send a request to the cache proxy with the HTTP method ``PURGE`` (using
 the word "PURGE" is a convention, technically this can be any string) instead
 of ``GET`` and make the cache proxy detect this and remove the data from the
-cache instead of going to Symfony to get a response.
+cache instead of going to the application to get a response.
```
Comment (ifdattic) [inline book/http_cache.rst]: The topic mostly uses *application* and does tie itself to Symfony, as this part is more about general working of cache I think it's a good idea to also just use application

**Comment by ifdattic** [comment]
Line 1045, the PHP example doesn't have a great readability and in my opinion could use some improvements

**Comment by WouterJ** [comment]
Please note https://github.com/symfony/symfony-docs/pull/4779 , where I've updated the examples.

**Comment by WouterJ** [comment]
Closing in favor of #4861

**Pull request closed without merging**